common/components/card/template.njk

```diff
@@ -42,7 +42,7 @@
           {% if item.label and (item.html or item.text) %}
             <div class="app-card__meta-list-item">
               <dt class="app-card__meta-list-key{% if item.hideLabel %} govuk-visually-hidden{% endif %}">
-                {{ item.label }}
+                {{ item.label | safe }}
```

Comment:
It would be nice to support the pattern used elsewhere with `.text` and `.html` to support unescaped values. But I'm not sure all the places this is being used and whether it would be trickier to change?

Reply:
I agree that using `.label.text` and `.label.html` would be nicer, from a quick search though it looks like that'd result in over 170 changes, maybe something for the future instead?

Reply:
Hmm, are all those instances using this particular template? I think the other instances of `label` within a component will all be escaping by default like this one would have been.

I think my concern was more that we're making all instance now unescaped.

From what I can see we'd only need to update it in these locations:
- [court-case-to-card-component](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/0adb469d7a6508dfb7e79f3b394bb4aab1b9834a/common/presenters/court-case-to-card-component.js#L14)
- [profile-to-card-component](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/0adb469d7a6508dfb7e79f3b394bb4aab1b9834a/common/presenters/profile-to-card-component.js#L43)

I'm happy to leave this for now though but maybe at a GH issue to keep track?